tor_ocr/main.py

```diff
@@ -268,12 +301,25 @@ def noop(cfg):
     logging.info('Loop!')
 
 
+def get_volunteer_id(reddit_username):
+    #remove first 2 characters (eg: u/transcribot => transcribot)
+    formatted_name = reddit_username[2:]
```

Comment:
Is it guaranteed that we'll always get a string where the first two characters are `u/`? Let's add a quick check for that; if we also change the arg from `reddit_username` to `username` for length purposes, we can fit a ternary to take care of it:

```suggestion
    formatted_name = reddit_username[2:] if username.startswith("u/") else username
```

Reply:
I like it, updating!